iota/Subscriber.go

```diff
@@ -47,8 +53,18 @@ func NewSubscriber(nodeConfig *configfile.NodeConfig, subConfig *configfile.SubC
 	C.sub_receive_announce(sub.Subscriber, address)
```

Comment:
After incorporating changes and using the v1.1.0 .so that you and I built earlier this week, I am receiving the following error on this line.
```
{"timestamp":"2021-07-15T16:39:20Z","hostname":"ubuntu","application":"./alvarium-example","line-number":"publisher.go:198","log-level":"debug","message":"announcement response - { \"announcement_id\": \"32bc4e28380e08ed4923b515481b045a028662441e7f28c2da82d7b838cc784d0000000000000000:6936d4e499c982365578a582\" }"}
{"timestamp":"2021-07-15T16:39:20Z","hostname":"ubuntu","application":"./alvarium-example","line-number":"sdk.go:41","log-level":"error","message":"sub_receive_announce() returned error \nSTREAMS ERROR: Operation failed to execute properly"}
```
I notice in the PR there's no error checking here so I removed my error check to see if it would fall through and work but no luck.